packages/teams-js/package.json

```diff
@@ -11,7 +11,7 @@
   "main": "./dist/MicrosoftTeams.min.js",
   "typings": "./dist/MicrosoftTeams.d.ts",
   "scripts": {
-    "build": "yarn lint && webpack && yarn docs:validate",
+    "build": "yarn lint && webpack && yarn docs:validate && node updateBlazorApp.js",
```

Comment:
Blazor App is a test app why the need of `updateBlazorApp` here?

Reply:
see mega-comment